graphics/rcd/lang/en_GB.txt

```diff
@@ -70,8 +70,18 @@ strings {
 		QUIT_YES:     "Yes";
 
 		// Error message strings.
-		ERROR_MESSAGE_CAPTION: "Message";
-		ERROR_MESSAGE_SPRITE:  "Failed to draw sprite";
+		ERROR_MESSAGE_CAPTION:        "Message";
+		ERROR_MESSAGE_SPRITE:         "Failed to draw sprite";
```

Comment:
This looks like a weird message. It sounds like a technical thing rather than a user problem.
How is a chief executive of an amusement park going to solve this problem?

Arguably, we should have a different way to deal with these things?